preview/containers/content/index.es

```diff
@@ -56,6 +82,20 @@ class Inputs extends PureComponent {
     console.warn('Selected emoji:', list[key]);
   }
 
+  handleTagsPickerSelect(tag) {
+    const value = this.inputRef.current.getValue().concat([tag]);
```

Comment:
why do u need this?

Reply:
For a proper example of tag cloud population with new tags.

Reply:
no need, for tag cloud receives only array of items.